rules/how-to-name-documents/rule.md

```diff
@@ -27,13 +27,19 @@ When naming documents and images, use **descriptive words** and **kebab-case** 
 
 <!--endintro-->
 
-## ✅ Use relevant words
+## Naming files
+
+### ✅ Use relevant words
```

Comment:
**question**: Is this sub-section required, given that it is only 1 in its section?

Reply:
I added an extra tip for naming files, and an intro to that H2